config/environments/test.rb

```diff
@@ -10,7 +10,7 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  config.eager_load = true # set eager loading to true so that ALL files are reported by simplecov, not just those that are loaded
```

Comment:
The default in Rails is something like this now:

```ruby
config.eager_load = ENV["CI"].present?
```

In my opinion that's preferable to eager loading every time anyone wants to run a test.

What do you think?

Reply:
Much better